diff -Nur Fop-0.20.3rc.orig/build.sh Fop-0.20.3rc/build.sh
--- Fop-0.20.3rc.orig/build.sh	Sat Jan 19 17:15:36 2002
+++ Fop-0.20.3rc/build.sh	Wed Jan 30 20:04:14 2002
@@ -13,7 +13,8 @@
   exit 1
 fi
 LIBDIR=lib
-LOCALCLASSPATH=$JAVA_HOME/lib/tools.jar:$JAVA_HOME/lib/classes.zip:$LIBDIR/ant.jar:$LIBDIR/batik.jar:$LIBDIR/buildtools.jar:$LIBDIR/xerces-1.2.3.jar:$LIBDIR/xalan-2.0.0.jar:$LIBDIR/xalanj1compat.jar:$LIBDIR/bsf.jar
+JCD=/usr/lib/java
+LOCALCLASSPATH=$JAVA_HOME/lib/tools.jar:$JAVA_HOME/lib/classes.zip:$LIBDIR/ant.jar:$JCD/batik.jar:$JCD/xml-apis.jar:$LIBDIR/buildtools.jar:$JCD/xerces.jar:$JCD/xalan.jar:$LIBDIR/xalanj1compat.jar:$LIBDIR/bsf.jar
 LOCALCLASSPATH=$LOCALCLASSPATH:$LIBDIR/jimi-1.0.jar:$LIBDIR/logkit-1.0b4.jar:$LIBDIR/avalon-framework-4.0.jar
 
 ANT_HOME=$LIBDIR
